lib/bundles/inspec-compliance/target.rb

```diff
@@ -24,11 +25,23 @@ def self.resolve(target)
 
       # check if we have a compliance token
       config = Compliance::Configuration.new
-      return nil if config['token'].nil?
+      if config['token'].nil?
+        Inspec::UI.exit_with_error <<EOF
```

Comment:
Its a great idea to add better information about an error, but that should be encapsulated in exceptions